lib/templates-sync.php

```diff
@@ -26,7 +26,7 @@ function _gutenberg_create_auto_draft_for_template( $post_type, $slug, $theme, $
 		array(
 			'post_type'      => $post_type,
 			'post_status'    => array( 'publish', 'auto-draft' ),
-			'title'          => $slug,
+			'name'           => $slug,
```

Comment:
This change is needed because we want to search templates by `slug` (called `name` in WP_Query) rather than `title` that can be customized.